addons/web/static/src/core/domain.js

```diff
@@ -291,6 +291,11 @@ function matchCondition(record, condition) {
 
     const fieldValue = typeof field === "number" ? field : record[field];
     switch (operator) {
+        case "=?":
```

Comment:
not sure to understand the point of this operator. Is there a task? Commit message should explain a bit